packages/phishing-controller/src/PhishingController.ts

```diff
@@ -45,6 +46,46 @@ export interface EthPhishingDetectConfig {
   version: number;
 }
 
+/**
+ * @type PhishingStalelist
+ *
+ * Interface defining expected type of the stalelist.json file.
+ * @property allowlist - List of approved origins (legacy naming "whitelist")
+ * @property blocklist - List of unapproved origins (legacy naming "blacklist")
+ * @property fuzzylist - List of fuzzy-matched unapproved origins
+ * @property tolerance - Fuzzy match tolerance level
+ * @property lastUpdated - Timestamp of last update.
+ */
+export interface PhishingStalelist {
+  allowlist: string[];
+  blocklist: string[];
+  fuzzylist: string[];
+  tolerance: number;
+  name: string;
```

Comment:
Hmm, this property appears to be missing from the data returned from the endpoint, and from the JSDoc comment.

We used to use this property to know which list was responsible for each block. But it seems that that won't be possible anymore with this combined format.

Reply:
We discussed this on Slack, and decided to drop this name. It would take too long to rework the API to preserve this data, and it would ultimately increase costs.

Instead we can update the phishing warning page to dynamically grab the latest config file, and check whether a given block is from us or Phishfort. I'll follow up on this elsewhere.

cc @409H